crm_action/__openerp__.py

```diff
@@ -1,15 +1,20 @@
 # -*- coding: utf-8 -*-
-# © 2015-2016 Savoir-faire Linux (<http://www.savoirfairelinux.com>)
+# Copyright 2015-2016 Savoir-faire Linux (<http://www.savoirfairelinux.com>)
+# Copyright 2017 Tecnativa - Vicent Cubells
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     'name': 'CRM Action',
-    'version': '8.0.1.2.0',
-    'author': 'Savoir-faire Linux,Odoo Community Association (OCA)',
+    'version': '9.0.1.0.0',
+    'author': 'Savoir-faire Linux, '
+              'Tecnativa, '
```

Comment:
I actually agree with you that some contributions don't deserve a place here. A migration always deserve it? Well, it depends. If you only follow the migration guide, and throw it here, maybe not. It even requires the check through the UI or unit tests... so it's not easy to set a general rule.